osidb/models/tests/test_affect.py

```diff
@@ -80,3 +80,18 @@ def test_resolved_dt_auto_update(
         affect.save()
         assert affect.is_resolved == is_resolved_2
         assert affect.resolved_dt or not is_resolved_2
+
+    @pytest.mark.parametrize(
+        "purl,ps_component",
+        [
+            (
+                "pkg:rpm/redhat/nginx@1.14.1-9.module+el8.0.0+4108+cba21616?arch=x86_64&rpmmod=mainstream",
+                "mainstream/nginx",
+            )
+        ],
+    )
+    def test_purls(self, purl, ps_component):
```

Comment:
I think `test_purls` is a bit vague as the test itself tests the correct inferring of the `ps_component` from `purl`. Maybe `test_ps_component_from_purl` or something like that?